### DISCUSSION TITLE
CartPole Emitter-Receiver Scheme Tutorial

### DISCUSSION THREAD

**Discussion by startgram** [aidudezzz/deepbots-tutorials#35] [Q&A]
Привет! Большое спасибо за вашу работу! Я начинаю только эту тему, и у меня есть несколько ошибок в моей консоли:
УСТАРЕВАНИЕ: Robot.getEmitter устарел, используйте вместо него Robot.getDevice.
УСТАРЕВАНИЕ: Robot.getReceiver устарел, используйте вместо него Robot.getDevice.
Трассировка (последний последний вызов):
Файл «supervisorController.py», строка 93, в
supervisor = CartPoleSupervisor()
Файл «supervisorController.py», строка 14, в init
self.respawnRobot()
Файл «supervisorController.py», строка 32 , in respawnRobot
childrenField.importMFNode(-2, "CartPoleRobot.wbo") # Загрузить робота из файла и добавить в предпоследнюю позицию
AttributeError: 'Поле' объект не имеет атрибута '
ВНИМАНИЕ: контроллер 'supervisorController' вышел со статусом: 1.

**Comment by tsampazk** [comment]
Hey there, as far as i can tell this is the same question as the one posted in the emitter receiver comments? I answered there.

**Reply by startgram** [reply]
Hi! Thank you for your answer! Yes, i'm trying second tutorial without reseiver at the present.
